addons/rating/controllers/main.py

```diff
@@ -25,6 +25,14 @@ def action_open_rating(self, token, rate, **kwargs):
             raise ValueError(_("Incorrect rating: should be 1, 3 or 5 (received %d)"), rate)
 
         rating, record_sudo = self._get_rating_and_record(token)
+        session_partner = request.env.user.partner_id
+        if rating.partner_id != session_partner and \
```

Comment:
I don't think we need to check if the person is in the followers, can't we just do:

`if not request.env.user._is_public() and request.env.user.partner_id != rating.partner_id`

(That way the customer can still access the page when not logged in).

?